.gitignore

```diff
@@ -5,3 +5,4 @@ nsis/nsis/
 .DS_Store
 .ionide
 report*.json
+node_modules/
```

Comment:
Can you add an extra line break at the end of this file for consistency?

Reply:
I've added it manually, maybe consider using `.editorconfig`?

Reply:
It looks like the line break may not have been added, or maybe the commit wasn't pushed.

Yup! I'll add a `.editorconfig` separately as to avoid having merge conflicts since so much was moved around on this PR.